Public/New-HTMLSection.ps1

```diff
@@ -122,7 +123,7 @@ Function New-HTMLSection {
                 New-HTMLAnchor -Id "hide_$RandomNumber" -Href 'javascript:void(0)' -OnClick "hide('$RandomNumber');" -Style $HideStyle -Text '(Hide)'
             }
             New-HTMLTag -Tag 'div' -Attributes @{ class = $ClassName; id = $RandomNumber; Style = $HiddenDivStyle } -Value {
-                New-HTMLTag -Tag 'div' -Attributes @{ class = "$ClassName collapsable"; id = $RandomNumber; } -Value {
+                New-HTMLTag -Tag 'div' -Attributes @{ class = "$ClassName collapsable"; id = $RandomNumber; Style = "justify-content: $JustifyContent";  } -Value {
```

Comment:
from what I remember if you will use `Style = @{ justify-content = $justifyContent }` it will convert this properly to HTML/CSS and if the $JustifyContent will be null/empty it will get removed during conversion. This allows you to build complicated styles without much care for non-defined parameters.

Reply:
That's right! It does remove it if the parameter is not selected.